tsdb/index/postings_test.go

```diff
@@ -950,7 +950,7 @@ func TestMemPostingsStats(t *testing.T) {
 	p.Add(2, labels.FromStrings("label", "value1"))
 
 	// call the Stats method to calculate the cardinality statistics
-	stats := p.Stats("label", 10)
+	stats := p.Stats("label", 10, func(name, value string, n uint64) uint64 { return uint64(len(name)+len(value)) * n })
```

Comment:
why not using a "real" `SizeOfLabels`, `stringlabels`'s as that's the default?

Reply:
We run the tests with 3 different `-tags`; they would need to have 3 sets of results to pass.

Reply:
Ah, I see. It would be great to have this as a comment so future readers don’t have to scratch their heads. However, if they're curious enough, they'll find the answer here. ;)